### PR TITLE
toggle anon zaps

### PR DESCRIPTION
**Pull request by futurepaul**
getting close on ui / logic but the zaps don't seem to be happening @benthecarman

### CODE REVIEW

**Comment by benthecarman** [comment]
Fixed the zaps, was a problem with https://github.com/MutinyWallet/cloudflare-lnurl

**Comment by benthecarman** [comment]
When I was testing, noticied some weirdness with the confirm send button not being enabled when it should

File: src/routes/Send.tsx [hunk 1]
```diff
@@ -569,10 +553,15 @@ export function Send() {
...
+                        ? contact()?.npub
+                        : undefined;
+                console.log("zapnpub", zapNpub);
                 const payment = await state.mutiny_wallet?.lnurl_pay(
                     lnurlp()!,
                     amountSats(),
-                    undefined, // zap_npub
+                    zapNpub, // zap_npub
                     tags,
                     undefined // comment
```
Comment (benthecarman) [inline src/routes/Send.tsx:566]: if it's an anon zap, should we set the comment to `whatForInput()`?

Comment (futurepaul) [reply src/routes/Send.tsx]: I added this but I don't know how to test if the note is getting through

**Review by benthecarman — APPROVED** [review]
works for me

**Comment by TonyGiorgio** [comment]
Is this good to merge?

**Comment by futurepaul** [comment]
yeah just retested it looking good gonna merge

**Pull request merged**